albumentations/augmentations/transforms.py

```diff
@@ -846,7 +846,7 @@ def apply(self, img, crop_height=0, crop_width=0, h_start=0, w_start=0, interpol
 
     def get_params_dependent_on_targets(self, params):
         img_h, img_w = params['image'].shape[:2]
-        if 'bboxes' not in params:  # less likely, this class is for use with bboxes.
+        if len(params['bboxes']) == 0:  # less likely, this class is for use with bboxes.
```

Comment:
if `bboxes` not in params this code will throw 'KeyError' so we can check both conditions

Reply:
https://github.com/albu/albumentations/blob/d85bab59eb8ccb0a2fec86750f94173e18e86395/albumentations/core/transforms_interface.py#L56-L58

if `bboxes` is not given, line 57 will throw 'KeyError', before enter the `get_params_dependent_on_targets` method.

Reply:
@albu This transform works only with bboxes, so if we get `KeyError` it will be right behavior

Reply:
Then we should assert that this transform does not work without bboxes.

Reply:
@albu Done.